virtualization-installs/jenkins-install.sh

```diff
@@ -62,7 +62,8 @@ fi
 # sudo systemctl start jenkins
 
 
-# # Output the initial admin password
+# We are going to have to probably do a yml file to pass this password in.
+# Output the initial admin password
 # echo "Initial admin password:"
```

Comment:
@stephenlauck I am not sure how to standard out the initial password?
- Do i use a yaml file?
- Or just `echo "Initial admin password:" > ~./some-file/`?

Reply:
I would just cat the file they say and have it output at the end of the script like you have here.